src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -364,12 +361,13 @@ public ModelingSubmissionResource(SubmissionRepository submissionRepository, Res
         participation.setResults(null);
 
         // do not send the result to the client if the assessment is not finished
-        if (modelingSubmission.getResult() != null && (modelingSubmission.getResult().getCompletionDate() == null || modelingSubmission.getResult().getAssessor() == null)) {
-            modelingSubmission.setResult(null);
+        if (modelingSubmission.getLatestResult() != null
+                && (modelingSubmission.getLatestResult().getCompletionDate() == null || modelingSubmission.getLatestResult().getAssessor() == null)) {
+            modelingSubmission.setResults(new ArrayList<Result>());
```

Comment:
Ì just want to make sure that `setResults` is correct and not `replaceLatestOrIfEmptyAddResult` because the deleted line calls `setResult`